pkg/cmd/pulumi/deployment.go

```diff
@@ -165,12 +188,54 @@ func newDeploymentSettingsCmd() *cobra.Command {
 	cmd.AddCommand(newDeploymentSettingsPullCmd())
 	cmd.AddCommand(newDeploymentSettingsUpdateCmd())
 	cmd.AddCommand(newDeploymentSettingsDestroyCmd())
+	cmd.AddCommand(newDeploymentSettingsEnvCmd())
+	cmd.AddCommand(newDeploymentSettingsSetCmd())
 
 	return cmd
 }
 
+func initializeDeploymentSettingsCmd(cmd *cobra.Command, stack string) (context.Context, *display.Options,
+	backend.Stack, *workspace.ProjectStackDeployment, backend.Backend, error,
```

Comment:
Should this return a struct and an error to avoid the complicated return and relying on sequence?